SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -425,6 +425,28 @@ FHandoverChangeState USpatialActorChannel::CreateInitialHandoverChangeState(cons
 	return HandoverChanged;
 }
 
+void USpatialActorChannel::UpdateVisibleComponent(AActor* InActor)
+{
+	// Make sure that the InActor is not a PlayerController, GameplayDebuggerCategoryReplicator or GameMode.
+	if (SpatialGDK::IsActorClassAlwaysRelevant(InActor))
```

Comment:
Was my previous comment of 

> I wonder if we should call NetDriver->RefreshActorVisibility(InActor, true);
That would be based on if we are able to change bAlwaysRelevant at runtime or not. If we can change it after the actor has spawned, then it's possible to make a non-hidden actor, then mark it as hidden, then mark it as always relevant... Perhaps, worth a quick check.

addressed? I'm still under the impression we should move this early out into the code which attempts to make an actor not visible ```
NetDriver->RefreshActorVisibility(InActor, false);
```

Reply:
Ah I see that `IsActorClassAlwaysRelevant` isn't actually based around `bAlwaysRelevant` but rather the particular NetRelevancy checks for particular classes. I find the name a little confusing then in this case since I assumed it was also doing the role of checking if the actor has `bAlwaysRelevant`...

Perhaps we should change this method name to `SpatialGDK::DoesActorClassIgnoreVisibilityCheck` or something more explicit.